episodes/02-setup.md

```diff
@@ -164,10 +370,14 @@ issues, it's safer to view the configuration with:
 $ git config --list
 ```
 
+::::::::::::::::::::::::::::::
+-->
+
 And if necessary, change your configuration using the
-same commands to choose another editor or update your email address.
+same commands to update your email address.
 This can be done as many times as you want.
 
+
 :::::::::::::::::::::::::::::::::::::::::  callout
 
 ## Proxy
```

Comment:
This section feels isolated. I have never used a proxy for Git, so I do not fully understand when one would be necessary. Is this information critical for new Git users? If not I would remove it. If it is crucial information I would explain when this step would be required in more detail.

Reply:
I agree, I opted remove it's visibility https://github.com/epiverse-trace/git-rstudio-basics/pull/13/commits/f7bf947f6c2967e774c1a983e26f9a78288e9925